src/components/TldrawCanvas.tsx

```diff
@@ -945,13 +1017,19 @@ const TldrawCanvas = ({ title }: Props) => {
         );
       if (!validChanges.length) return;
       clearTimeout(serializeRef.current);
-      serializeRef.current = window.setTimeout(() => {
+      serializeRef.current = window.setTimeout(async () => {
         const state = _store.serialize();
         const props = getBlockProps(pageUid) as Record<string, unknown>;
         const rjsqb =
           typeof props["roamjs-query-builder"] === "object"
             ? props["roamjs-query-builder"]
             : {};
+        // we need this bc Roam doesn't update edit/user or edit/time when we just edit block/props
```

Comment:
Josh has acknowledged this as a bug

Reply:
Looks like `edit/time` is updating now.  But not `edit/user`.
Tested and ref: https://roamresearch.com/#/app/help/page/O3JfglQsT